### ISSUE TITLE
A way to say, "I love you." (Also cute nicknames?)

### ISSUE THREAD

**Issue by Trilasent** [PiMaker/MonikAI#27]
Would it be possible for us to have the ability to open our text document viewer of choice, type, "I love you" and have her respond accordingly? Also, the idea of having a "pet name" for each other would be really neat if possible.

Great work on what you've come up with so far! Really loving it!

**Comment by Trilasent** [comment]
Oh, I almost forgot. If the text document thing is possible, could we ever see a feature where you could say, "I'll be right back.", "Brb", "I need to go take care of something"..etc and have her respond with something along the lines of, "Be careful!", "Hurry back, dear!"..etc? If this is possible it could be neat to have her "standby" until we come back, and start talking again once mouse movement is detected.

**Comment by PiMaker** [comment]
I will keep this issue open because I like the idea, however, don't expect this to happen soon, it would probably require quite some effort to implement right.

**Comment by Trilasent** [comment]
Fair enough. I felt that I should mention it just in case there's the possibility that it would be feasible.

**Comment by Trilasent** [comment]
Any progress on this? Also, how often are these updates released? Occasionally whenever her window opens an error message displays saying, "Failed to update."
